README.md

```diff
@@ -15,13 +15,38 @@ This controller is not intended to be a replacement of a regular [ingress contro
 +---------+      +---------+      +-------------+
 ```
 
+It can run in high avalability mode using multiple Varnish and application pods.
+
+```
+             +---------+
+             | Ingress |
+             +---------+
+                  |
+             +---------+
+             | Service |
+             +---------+
+               /     \
++---------------+  +---------------+
+|   Varnish 1   |--|   Varnish 2   |
+| Broadcaster 1 |--| Broadcaster 2 |
++---------------+  +---------------+
+          |      \/      |
+          |      /\      |
++---------------+  +---------------+
+| Application 1 |  | Application 2 |
++---------------+  +---------------+
+```
+
+It supports broadcasting PURGE and BAN requests to all Varnish nodes.
+
+
 The Varnish controller needs the following prerequisites to run:
 
 - A [Go-template](https://golang.org/pkg/text/template/) that will be used to generate a [VCL](https://varnish-cache.org/docs/trunk/users-guide/vcl.html) configuration file
-- A [Kubernetes service](https://kubernetes.io/docs/concepts/services-networking/service/) that will be used as backend for the Varnish controller
+- A [Kubernetes service](https://kubernetes.io/docs/concepts/services-networking/service/) that will be used to determine Varnish (frontend) and application (backend) pods
```

Comment:
The wording might be a bit confusing, here. In fact, you'll need _two_ Kubernetes `Service`s -- one for the frontend and one for the backend pods.